ert_shared/dark_storage/enkf.py

```diff
@@ -46,17 +34,19 @@ def get_name(type, uuid):
 
 
 def get_res(*, _: None = Depends(security)) -> LibresFacade:
-    global _libres_facade
-    if _libres_facade is None:
-        init_facade()
-    return _libres_facade
+    configfile = os.environ["ERT_STORAGE_RES_CONFIG"]
+    config = ResConfig(configfile)
+    os.chdir(config.config_path)
+    ert = EnKFMain(config, strict=True)
+    notifier = ErtNotifier(ert, config)
+    adapter = ErtAdapter()
+    with adapter.adapt(notifier):
+        yield adapter.enkf_facade
```

Comment:
Does this mean that we are going to initialize a new instance of EnkMain every time a new request is handled by dark_storage?

Reply:
i haven't thought about this, removing my approval until we have clarified

Reply:
Undid these changes